datasets/varen/varen.json

```diff
@@ -0,0 +1,175 @@
+{
+  "type": "dataset",
+  "id": "Varen",
+  "title": "Varen",
+  "description": "Inforatie over de over de pleziervaart en het ligplaatsen register voor de pleziervaart.",
+  "dateCreated": "2020-08-01T00:00:00",
+  "dateModified": "2020-08-01T00:00:00",
+  "license": "Creative Commons, Naamsvermelding",
+  "homepage": "https://data.amsterdam.nl",
+  "language": "nl",
+  "status": "beschikbaar",
+  "version": "0.0.1",
+  "objective": "Programma varen heeft de verplichting om openbare informatie waaronder op en afstap plaatsen te publiceren voor de bewoners van Amsterdam en de ketenpartners.",
+  "temporalUnit": "nvt",
+  "contactPoint": {
+    "name": "",
+    "email": "datapunt@amsterdam.nl"
+  },
+  "accrualPeriodicity": "maandelijks",
+  "spatialDescription": "Gemeente Amsterdam",
+  "theme": "Verkeer",
+  "publisher": "OIS",
+  "owner": "Gemente Amsterdam, Toezicht en Handhaving Openbare Ruimte",
+  "authorizationGrantor": "Toezicht en Handhaving Openbare Ruimte",
+  "keywords": "ligplaatsenregister, varen, opstaplocatie, afstaplocatie, grachten, varen",
+  "hasBeginning": "2020-01-01T00:00:00",
+  "crs": "EPSG:28992",
+  "tables": [
+    {
+      "id": "var_opafstapplaats",
+      "type": "table",
+      "provenance": "var_opafstapplaats",
+      "title": "var_opafstapplaats",
+      "description": "Deze dataset bevat de gegevens over op- en afstaplocaties voor pleziervaart, passagiersvaart en transport over het water. De gemeente wil daarbij voldoende ruimte voor bewoners creëren, onder andere doorbeheersing van het aantal vaarbewegingen en het spreiden van plezier- en passagiersvaart.",
+      "schema": {
+        "required": ["volgnr"],
+        "mainGeometry": "geometrie",
+        "display": "volgnr",
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "id": {
+            "type":"string",
+            "provenance": "var_opafstapplaats_id",
+            "description": "Uniek identificerend kenmerk van het record"  
+          },
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@1.1.1#/definitions/schema"
+          },
+          "volgnr": {
```

Comment:
Optie: Je kunt er voor kiezen om volgnr voluit te schrijven. En de in provenance van het veld "volgnr" te zetten.